assets/javascripts/modules/checkout/omniture.js

```diff
@@ -0,0 +1,48 @@
+define(['$'], function ($) {
+    'use strict';
+
+    var omniture = {};
+
+    function getSubscriptionInfo(){
+        var selectedFrequency = $('.js-option-mirror-group input:checked');
```

Comment:
I'd recommend using a different `.js-` class to avoid coupling this to the `optionMirror` component.